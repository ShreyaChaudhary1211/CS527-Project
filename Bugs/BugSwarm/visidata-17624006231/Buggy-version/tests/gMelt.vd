sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Unit		addcol-expr	'$'	=	
benchmark	'$'		rename-col	Unit_Currency	^	
benchmark	Unit_Currency		freeze-col		'	
benchmark	Unit_Currency		hide-col		-	
benchmark	Unit_Currency_frozen		rename-col	Unit_Currency	^	
benchmark	Unit	0	split-col	[$â‚ª]	:	
benchmark	Unit		hide-col		-	
benchmark	Unit_re0		hide-col		-	
benchmark	Unit_re1		rename-col	Unit	^	
benchmark	Unit_Currency	7	edit-cell	AUD$	e	
benchmark	Unit_Currency	12	edit-cell	EUR	e	
benchmark	Unit_Currency	17	edit-cell	PES	e	
benchmark	Unit_Currency		addcol-expr	'$'	=	
benchmark	'$'		rename-col	Total_Currency	^	
benchmark	Date		key-col		!	
benchmark	Customer		key-col		!	
benchmark	SKU		key-col		!	
benchmark	Paid		rename-col	Total_Cost	^	
benchmark	Unit		rename-col	Unit_Cost	^	
benchmark			melt-regex	(.*)_(.*)	gM	
benchmark_melted			columns-sheet		C	
benchmark_melted_columns	name		sort-desc		]	
benchmark_melted_columns			quit-sheet		q	
