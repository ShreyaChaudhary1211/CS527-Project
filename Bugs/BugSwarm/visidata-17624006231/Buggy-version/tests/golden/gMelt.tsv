Date	Customer	SKU	Variable	Currency	Cost
7/3/2018 1:47p	Robert Armstrong	FOOD213	Unit	$	12.95
7/3/2018 1:47p	Robert Armstrong	FOOD213	Total	$	$51.8
7/3/2018 3:32p	Kyle Kennedy	FOOD121	Unit	$	4.22
7/3/2018 3:32p	Kyle Kennedy	FOOD121	Total	$	$4.22
7/5/2018 4:15p	Douglas "Dougie" Powers	FOOD121	Unit	$	4.22
7/5/2018 4:15p	Douglas "Dougie" Powers	FOOD121	Total	$	$4.22
7/6/2018 12:15p	桜 高橋 (Sakura Takahashi)	FOOD122	Unit	$	1.29
7/6/2018 12:15p	桜 高橋 (Sakura Takahashi)	FOOD122	Total	$	157¥
7/10/2018 10:28a	David Attenborough	NSCT201	Unit	$	.05
7/10/2018 10:28a	David Attenborough	NSCT201	Total	$	$1.5
7/10/2018 5:23p	Susan Ashworth	CAT060	Unit	$	720.42
7/10/2018 5:23p	Susan Ashworth	CAT060	Total	$	$720.42
7/10/2018 5:23p	Susan Ashworth	FOOD130	Unit	$	14.94
7/10/2018 5:23p	Susan Ashworth	FOOD130	Total	$	$14.94
7/13/2018 10:26a	Wil Wheaton	NSCT523	Unit	AUD$	39.95
7/13/2018 10:26a	Wil Wheaton	NSCT523	Total	$	$39.95
7/13/2018 3:49p	Robert Armstrong	FOOD216	Unit	$	12.95
7/13/2018 3:49p	Robert Armstrong	FOOD216	Total	$	$51.8
7/17/2018 9:01a	Robert Armstrong	FOOD217	Unit	$	12.95
7/17/2018 9:01a	Robert Armstrong	FOOD217	Total	$	$51.8
7/17/2018 11:30a	Helen Halestorm	LAGO342	Unit	$	32.94
7/17/2018 11:30a	Helen Halestorm	LAGO342	Total	$	$65.88
7/18/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Unit	$	1.29
7/18/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Total	$	157¥
7/19/2018 10:28a	Rubeus Hagrid	FOOD170	Unit	EUR	44.95
7/19/2018 10:28a	Rubeus Hagrid	FOOD170	Total	$	$224.75
7/20/2018 2:13p	Jon Arbuckle	FOOD167	Unit	$	3.95
7/20/2018 2:13p	Jon Arbuckle	FOOD167	Total	$	$197.5
7/23/2018 1:41p	Robert Armstrong	FOOD215	Unit	$	12.95
7/23/2018 1:41p	Robert Armstrong	FOOD215	Total	$	$51.8
7/23/2018 4:23p	Douglas "Dougie" Powers	TOY235	Unit	$	16.12
7/23/2018 4:23p	Douglas "Dougie" Powers	TOY235	Total	$	$16.12
7/24/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Unit	$	1.29
7/24/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Total	$	157¥
7/26/2018 4:39p	Douglas "Dougie" Powers	FOOD420	Unit	PES	15.70
7/26/2018 4:39p	Douglas "Dougie" Powers	FOOD420	Total	$	$15.7
7/27/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Unit	$	1.29
7/27/2018 12:16p	桜 高橋 (Sakura Takahashi)	FOOD122	Total	$	157¥
7/30/2018 12:17p	桜 高橋 (Sakura Takahashi)	RETURN	Unit	$	1.29
7/30/2018 12:17p	桜 高橋 (Sakura Takahashi)	RETURN	Total	$	157¥
7/31/2018 5:42p	Rubeus Hagrid	CAT060	Unit	$	720.42
7/31/2018 5:42p	Rubeus Hagrid	CAT060	Total	$	$3602.1
8/1/2018 2:44p	David Attenborough	FOOD360	Unit	$	5.72
8/1/2018 2:44p	David Attenborough	FOOD360	Total	$	$22.88
8/2/2018 5:12p	Susan Ashworth	CAT110	Unit	$	1,309.68
8/2/2018 5:12p	Susan Ashworth	CAT110	Total	$	$1309.68
8/2/2018 5:12p	Susan Ashworth	FOOD130	Unit	$	14.94
8/2/2018 5:12p	Susan Ashworth	FOOD130	Total	$	$44.82
8/6/2018 10:21a	Robert Armstrong	FOOD212	Unit	$	12.95
8/6/2018 10:21a	Robert Armstrong	FOOD212	Total	$	$51.8
8/7/2018 4:12p	Juan Johnson	REPT082	Unit	$	89.95
8/7/2018 4:12p	Juan Johnson	REPT082	Total	$	$89.95
8/7/2018 4:12p	Juan Johnson	RDNT443	Unit	$	1.49
8/7/2018 4:12p	Juan Johnson	RDNT443	Total	$	$1.49
8/10/2018 4:31p	Robert Armstrong	FOOD211	Unit	$	12.95
8/10/2018 4:31p	Robert Armstrong	FOOD211	Total	$	$51.8
8/13/2018 2:07p	Monica Johnson	RDNT443	Unit	$	1.49
8/13/2018 2:07p	Monica Johnson	RDNT443	Total	$	$1.49
8/13/2018 2:08p	María Fernández	FOOD146	Unit	$	2.00
8/13/2018 2:08p	María Fernández	FOOD146	Total	$	$4.0
8/15/2018 11:57a	Mr. Praline	RETURN	Unit	$	2300.00
8/15/2018 11:57a	Mr. Praline	RETURN	Total	$	-$2300.0
8/15/2018 3:48p	Kyle Kennedy	FOOD121	Unit	$	4.22
8/15/2018 3:48p	Kyle Kennedy	FOOD121	Total	$	$8.44
8/16/2018 11:50a	Helen Halestorm	RETURN	Unit	$	0
8/16/2018 11:50a	Helen Halestorm	RETURN	Total	$	$0.0
8/16/2018 4:00p	Kyle Kennedy	DOG010	Unit	$	2,495.99
8/16/2018 4:00p	Kyle Kennedy	DOG010	Total	$	$2495.99
8/16/2018 5:15p	Michael Smith	BIRD160	Unit	$	#ERR
8/16/2018 5:15p	Michael Smith	BIRD160	Total	$	$31.85
8/17/2018 9:26a	Rubeus Hagrid	NSCT201	Unit	$	.05
8/17/2018 9:26a	Rubeus Hagrid	NSCT201	Total	$	$0.25
8/20/2018 9:36a	Kyle Kennedy	RETURN	Unit	$	1,247.99
8/20/2018 9:36a	Kyle Kennedy	RETURN	Total	$	-$1247.99
8/20/2018 1:47p	מרוסיה ניסנהולץ אבולעפיה	GOAT224	Unit	$	499
8/20/2018 1:47p	מרוסיה ניסנהולץ אבולעפיה	GOAT224	Total	$	$160.51
8/20/2018 3:31p	Monica Johnson	NSCT201	Unit	$	.05
8/20/2018 3:31p	Monica Johnson	NSCT201	Total	$	$1.5
8/20/2018 5:12p	David Attenborough	NSCT084	Unit	$	.17
8/20/2018 5:12p	David Attenborough	NSCT084	Total	$	$5.10
8/21/2018 12:13p	Robert Armstrong	FOOD214	Unit	$	12.95
8/21/2018 12:13p	Robert Armstrong	FOOD214	Total	$	$51.8
8/22/2018 9:38a	David Attenborough	BIRD160	Unit	$	#ERR
8/22/2018 9:38a	David Attenborough	BIRD160	Total	$	$29.95
8/22/2018 2:13p	Jon Arbuckle	FOOD170	Unit	$	44.95
8/22/2018 2:13p	Jon Arbuckle	FOOD170	Total	$	$44.95
8/22/2018 5:49p	מרוסיה ניסנהולץ	SFTY052	Unit	$	61.70
8/22/2018 5:49p	מרוסיה ניסנהולץ	SFTY052	Total	$	$61.70
8/24/2018 11:42a	Robert Armstrong	FOOD218	Unit	$	12.95
8/24/2018 11:42a	Robert Armstrong	FOOD218	Total	$	$51.8
8/27/2018 3:05p	Monica Johnson	NSCT443	Unit	$	1.99
8/27/2018 3:05p	Monica Johnson	NSCT443	Total	$	$1.99
8/28/2018 5:32p	Susan Ashworth	CAT020	Unit	$	1,964.53
8/28/2018 5:32p	Susan Ashworth	CAT020	Total	$	$1964.53
8/28/2018 5:32p	Susan Ashworth	FOOD130	Unit	$	14.94
8/28/2018 5:32p	Susan Ashworth	FOOD130	Total	$	$29.88
8/29/2018 10:07a	Robert Armstrong	FOOD219	Unit	$	12.95
8/29/2018 10:07a	Robert Armstrong	FOOD219	Total	$	$51.8
8/31/2018 12:00a	Robert Armstrong	FOOD219	Unit	$	12.95
8/31/2018 12:00a	Robert Armstrong	FOOD219	Total	$	$1864.8
8/31/2018 5:57p	Juan Johnson	REPT217	Unit	$	99.95
8/31/2018 5:57p	Juan Johnson	REPT217	Total	$	$99.95
